crates/sui-framework/packages/sui-system/sources/sui_system.move

```diff
@@ -698,6 +698,13 @@ module sui_system::sui_system {
         self.set_epoch_for_testing(epoch_num)
     }
 
+    // NEVER remove `#[test_only]`
+    #[test_only]
+    public(package) fun increment_epoch_for_testing(wrapper: &mut SuiSystemState) {
```

Comment:
I think you can reuse the function above `set_epoch_for_testing`.

Reply:
i used this and decided with @emmazzz that this may be helpful when writing advancing-epoch tests. But you are right we could make the test work without this. I can remove it if you prefer